src/dawn/CodeGen/Cuda/MSCodeGen.hpp

```diff
@@ -107,6 +107,10 @@ class MSCodeGen {
   static std::string kBegin(const std::string dom, iir::LoopOrderKind loopOrder,
                             iir::Interval const& interval);
 
+  bool intervalPreviouslyAccessed(const int accessID, iir::Interval const& targetInterval,
```

Comment:
`T const&` again. I really think we should stick to one. 

If you feel like is is much more readable we can debate but since we once said we'd stick to the LLVM style, we should use `const T&` until we change that (or make an exception for this case). This is btw also the style that the cpp standard uses

Reply:
:+1: